pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -331,6 +331,20 @@ def make_package(args):
                 elif exists(join(args.private, "main.py")):
                     shutil.copyfile(join(args.private, "main.py"),
                                     join(main_py_only_dir, "main.py"))
+                if exists(join(args.private, "service", "main.pyo")):
```

Comment:
I would pull the repeated calls to `join` into a variable, since the arguments are the same each time.

Reply:
Ei...shouldn't be the file `main.pyc`?

`.pyo` was for python 2, but, as far as I know, for python 3 should be `.pyc`...since we are moving to python3...maybe we should change all this `.pyo` extensions to `.pyc`...or maybe set the file extension based on the python version used, to be compatible with both versions of python...

Reply:
Both `.pyc` and `.pyo` are handled now, thanks for pointing this out!